.claude/skills/error-handling-patterns/SKILL.md

```
@@ -39,115 +45,92 @@ When invoked:
-export class AppError extends Error {
-  constructor(
-    message: string,
-    public readonly code: string,
-    public readonly statusCode: number = 500,
-    public readonly details?: Record<string, unknown>
-  ) {
-    super(message);
-    this.name = "AppError";
-  }
-}
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)